components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml

```diff
@@ -702,27 +715,26 @@
 $xwiki.jsx.use('PhenoTips.PatientConsent')
 $xwiki.ssx.use('PhenoTips.PatientConsent')
 #if ($services.consentauthorizer.consentsGloballyEnabled())
-  (% id='consents-block' class='$xcontext.action' %)(((
+  #set($consentBaseUrl = "/rest/patients/${doc.getName()}/consents/")
+  (% id='consents-block' class='$xcontext.action' data-url='$consentBaseUrl' %)(((
     {{html clean='false' wiki='false'}}&lt;div class='patient-id hidden'&gt;${doc.getName()}&lt;/div&gt;{{/html}}
-    (% class='view-title hidden' %)(((
+    (% class='consents-title' %)(((
```

Comment:
Yep, it was on purpose. There weren't any styles associated with that class (nor `view-title` before). Nice spot, though!